### PR TITLE
Remove RedPhone code.

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @michaelkirk 

// FREEBIE

### CODE REVIEW

File: src/Messages/OWSMessageSender.m [hunk 1]
```diff
@@ -115,7 +115,7 @@ - (instancetype)initWithMessage:(TSOutgoingMessage *)message
     _successHandler = ^{
         typeof(self) strongSelf = weakSelf;
         if (!strongSelf) {
-            OWSAssert(NO);
+            OWSCAssert(NO);
```
Comment (charlesmchen) [inline src/Messages/OWSMessageSender.m]: Fixed a warning.

**Comment by michaelkirk** [comment]
LGTM

**Comment by charlesmchen** [comment]
@michaelkirk FYI not shipping this until the next release.

**Review by michaelkirk — APPROVED** [review]
LGTM

**Pull request merged**